_includes/components/hero.html

```diff
@@ -0,0 +1,46 @@
+<div class="hero">
+  <div class="container">
+
+    <!-- hero images-->
+    <div class="hero__img">
+      <img class="img-responsive hero__img--left" src="/img/{{include.img1}}" alt="iphone mockup">
```

Comment:
add hero__img to both hero__img--left and use it for styles that apply on both hero__img